src/Microsoft.AspNetCore.Rewrite/Internal/MatchResults.cs

```diff
@@ -12,5 +12,7 @@ public class MatchResults
 
         public GroupCollection BackReference { get; set; }
```

Comment:
Why do you still have the GroupCollection property? Wasn't BackReferenceCollection intended to replace this?